kratos/python/add_kratos_parameters_to_python.cpp

```diff
@@ -69,6 +69,10 @@ void  AddKratosParametersToPython(pybind11::module& m)
     .def(py::init<const std::string&>())
     .def(py::init<std::ifstream&>())
     .def(py::init<Parameters const&>())
+    .def("__copy__", [](const Parameters& self) {auto a = Kratos::make_shared<Parameters>(self); return a;})
+    .def("__copy__", [](const Parameters& self, py::object memo) {auto a = Kratos::make_shared<Parameters>(self); return a;})
+    .def("__deepcopy__", [](const Parameters& self) {auto a = Kratos::make_shared<Parameters>(); Parameters& r_a = *a;r_a = self; return a;})
+    .def("__deepcopy__", [](const Parameters& self, py::object memo) {auto a = Kratos::make_shared<Parameters>(); Parameters& r_a = *a;r_a = self; return a;})
```

Comment:
Why two times?

Reply:
Because is required by the copy library, which is the standard to use the copy in python

Reply:
ah so the second version (the one accepting a `py::object memo` object) is for the `copy` lib?

Reply:
```suggestion
    .def("__deepcopy__", [](const Parameters& self, py::object memo) {auto a = Kratos::make_shared<Parameters>(); Parameters& r_a = *a;r_a = self; return a;}) // used for the copy module
```